manual/src/html_processing/src/process_api.ml

```diff
@@ -129,6 +129,19 @@ let make_toc ~version ~search file config title body =
                            (manual_page_url ^ "/index.html")))
 
 
+let anchor_element node =
+  create_element ~class_:"anchor" ~attributes:["href", "#" ^ R.attribute "id" node] "a"
+
+let add_anchors_to_items body =
+  List.iter (fun kind ->
+      iter (fun node -> insert_before node (anchor_element node)) (body $$ (Printf.sprintf "span[id^=%S]" kind))
+    ) [ "VAL"; "TYPE"; "MODULE" ]
```

Comment:
We should add links to exceptions and extension constructor with `"EXCEPTION"` and `"EXTENSION"`.
We could also add `"METHOD"` and `"ATT"` to be exhaustive.

Reply:
Thanks, fixed.